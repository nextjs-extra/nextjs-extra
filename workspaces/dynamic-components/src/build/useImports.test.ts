import { useImports } from "./useImports";

test("useImports", () => {
  const files = [
    "node_modules/next/dynamic.js",
    "path/ComponentName.dynamic.jsx",
    "path/wrongComponentName.dynamic.tsx",
    "path/Wrong+Component:Name.dynamic.tsx",
  ];

  expect(useImports(files)).toMatchSnapshot();
});
