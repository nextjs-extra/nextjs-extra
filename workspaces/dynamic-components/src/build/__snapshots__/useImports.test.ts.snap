// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`useImports 1`] = `
"
  ComponentName: dynamic(() => import("./path/ComponentName.dynamic.jsx"), { suspense: true, loading: undefined }),"
`;
